cgfs.py

```diff
@@ -109,10 +107,9 @@ def setattr(self, key, value):
 
 
 class Directory(BaseFile):
```

Comment:
Should `FixedDirectory` also inherit from this class?

Reply:
I couldn't find a class named `FixedDirectory`?

Reply:
Sorry I meant `TempDirectory`.

Reply:
It already does...

Reply:
Hmm...